ti/testsplitter/consts.go

```diff
@@ -0,0 +1,14 @@
+package testsplitter
+
+const (
+	ApplicationName = "split-tests"
+	NumSplitsEnv    = "HARNESS_NODE_TOTAL" // Environment variable for total number of splits
+	CurrentIndexEnv = "HARNESS_NODE_INDEX" // Environment variable for the current index
+
+	SplitByFileTimeStr      = "file_timing"
+	SplitByClassTimeStr     = "class_timing"
+	SplitByTestcaseTimeStr  = "testcase_timing"
+	SplitByTestSuiteTimeStr = "testsuite_timing"
+	SplitByFileSizeStr      = "file_size"
+	SplitByTestCount        = "test_count"
```

Comment:
Is this testcase_count or file_count?

Reply:
`FileCount` will be implemented with CI Manager changes.